cw-relayer/config/config.go

```diff
@@ -137,6 +135,10 @@ func ParseConfig(configPath string) (Config, error) {
 		cfg.MaxTickTimeout = defaultTimeout.String()
 	}
 
+	if cfg.Keyring.PrivKey == "$PRIV_KEY" {
+		cfg.Keyring.PrivKey = key
+	}
```

Comment:
Can nested objects not be set via environment variables with viper?
I believe @zarazan might have done something on this